glusterd2/bricksplanner/subvoltype_replicate.go

```diff
@@ -44,3 +46,18 @@ func (s *replicaSubvolPlanner) BrickType(idx int) string {
 func init() {
 	subvolPlanners["replicate"] = &replicaSubvolPlanner{}
 }
+
+// ArbiterBrickSizeCalculation returns the arbiter brick size for that particular volume
+func ArbiterBrickSizeCalculation(brickSize uint64, avgFileSize uint64) (uint64, error) {
+	if avgFileSize == 0 {
```

Comment:
If avgFileSize is zero then consider default value `64K`

Reply:
As all the values are calculated in M, if we specify it in K then it will be ~0.064M which will be considered as zero because the size is an uint64. we need these to be changed once the default size is in bytes.

Reply:
define constant of 1M for now. Once this issue https://github.com/gluster/glusterd2/issues/1135 fixed, we can change that to 64K